test/FESpacesTests.jl

```diff
@@ -34,9 +34,15 @@ function main(parts,das)
   uh = interpolate(u,U)
   eh = u - uh
 
+  uh_dir = interpolate_dirichlet(u,U)
```

Comment:
Do these tests trigger all the different functions that you have added above? If not, I would add tests as per-required.

Reply:
Done in 7030e01a8373acb07da0d5d62a286a79984e0e51